apps/docs/src/app/core/component-docs/list/list-docs.component.ts

```diff
@@ -162,6 +163,14 @@ export class ListDocsComponent {
         }
     ];
 
+    listInteractive: ExampleFile[] = [
+        {
+            language: 'html',
+            code: listInteractiveHtml,
+            fileName: 'list-interactive-example'
```

Comment:
You are missing `component`

Reply:
It's not needed for `only-html` examples